src/server_manager/web_app/ui_components/outline-server-view.js

```diff
@@ -801,43 +803,61 @@ export class ServerView extends DirMixin(PolymerElement) {
     }
   }
 
-  _formatBytesTransferred(numBytes, emptyValue = '') {
+  _makeReadableDataAmount(amount, numDecimals) {
+    const readable = byte_size(amount, {precision: numDecimals});
+    return {
+      amount: readable.value,
+      // Make the unit singular
+      unit: readable.long.slice(0, -1)
```

Comment:
I don't think we should rely on this. `long` is supposed to be a huma-readable string. Will it always be an English string? It may change too. We should use the unit only, and map to Intl unit.

Alternatively, let's implement our own bytes formatter. The logic is quite simple.

Reply:
Done.  I got rid of our use of byte-size completely.